ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts

```diff
@@ -370,35 +369,40 @@ const triggerSet: TriggerSet<Data> = {
         'Specter of Zenos': 'spectre de Zenos',
       },
       'replaceText': {
+        '\\?': ' ?',
+        '\\(E->W\\)': '(E->O)',
+        '\\(SW->NW\\)': '(SO->NO)',
+        '\\(gun\\)': '(fusil)',
+        '\\(gun/spear\\)': '(fusil/lance)',
         '\\(spear\\)': '(lance)',
         'Antitwilight': 'Belle-de-nuit',
-        'Bright Blade': 'Lame blafarde',
+        'Bright Blade/Dark Blade': 'Lame blafarde/ténébreuse',
         'Concentrativity': 'Kenki concentré',
         'Crater': 'Explosion de fragment lunaire',
         'Dance Of The Dead': 'Danse des morts',
-        'Dark Blade': 'Lame ténébreuse',
         'Dispersivity': 'Onde Kenki',
         'Empire adds': 'Adds Impériaux',
         'Hagetsu': 'Pulvérisation lunaire',
         'Homeland adds': 'Adds Domiens',
-        'Lead Of The Underworld': 'Tir de l\'au-delà',
+        'Lead Of The Underworld(?!/Steel Of The Underworld)': 'Tir de l\'au-delà',
```

Comment:
Here I don't know if there is a code to have a shorter line. If so tell me and I would edit.

Reply:
maybe just `(?!/)` for this usage.
'cause there is only this example following a slash.
also the bottom one.

Reply:
```suggestion
        'Lead Of The Underworld(?!/)': 'Tir de l\'au-delà',
```

> Here I don't know if there is a code to have a shorter line. If so tell me and I would edit.
Maiko is suggesting this.  Does this work?